postgresStore.go

```diff
@@ -16,24 +16,21 @@ const (
 	UPSERT_TASK_SESSION string = `INSERT INTO task_session (username,taskid) VALUES ($1,$2) ON CONFLICT (username) DO UPDATE SET taskid=$2`
```

Comment:
I think I might name these constants like `SQLInsert`, `SQLDelete`, and so on. Prefixing the name with the _kind_ of constant it is helps when you have lots of different constants representing different kinds of things: the reader knows that anything starting `SQL...` is some SQL, for example.